tests/pf/test_pf_quadtree_inversion_linear.py

```diff
@@ -50,20 +50,15 @@ def create_mesh(h, padDist, nCpad, topo, data, min_height=1.0):
             padDist = np.ones((2, 2)) * 100
             nCpad = [2, 4]
 
-            mesh = mesh_builder_xyz(
+            self.mesh = mesh_builder_xyz(
                 topo[:, :2],
                 h,
                 padding_distance=padDist,
                 mesh_type="TREE",
             )
 
-            self.mesh = refine_tree_xyz(
-                mesh,
-                data[:, :2],
-                method="radial",
-                octree_levels=nCpad,
-                octree_levels_padding=nCpad,
-                finalize=True,
```

Comment:
@jcapriot, is there a replacement for `octree_levels_padding` in the new `refine_points` method? With the current change to the code, the method is not generating the same mesh as the original one.

Reply:
Actually for a one-to-one here, I think you need to do:
`padding_cells_by_level=[nCpad]`. The updated methods are bit more clear in what the arguments do compared to documentation of the old one.

Reply:
Well, it's been a while since I've looked at this, but I think you had the translation correctly to start. Like I said originally, the `octree_levels_padding` was unused for the radial option. The `refine_tree_xyz` function for radial basically looked at the `octree_levels` argument, then turned it into an array, iterated through it starting at the maximum level and created a ball around each point that was 

```Python
radius = 0
for i_level, n_pad in enumerate(octree_levels):
    radius += mesh.h[0].min() * n_pad * 2**(mesh.max_level - i_level) 
```

now it does:
``Python
radius = 0
h_min = np.max([h.min() for h in self.h])
for i_level, n_pad in enumerate(octree_levels):
    radius += h_min * n_pad * 2**(mesh.max_level - i_level)
```
So now it guarantees at least that many padding cells in all directions. So if the mesh had unequal cell sizes, it likely changed slightly.

Reply:
using `padding_cells_by_level=[nCpad]` falls into an error because `nCpad` is already a list. `padding_cells_by_level=nCpad` works fine, and today I'm noticing they are generating the same mesh. I'll give it one more try and see what happens. Ignore this comments for now 😅